packages/components/src/components/TokenGate/TokenGate.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext } from 'react';
+import { TokenGate } from '.';
+import { ethers } from 'ethers';
+import { NETWORKS, Provider, useWallet, Web3Context } from '@web3-ui/hooks';
+import { Text } from '@chakra-ui/layout';
+import { Button } from '@chakra-ui/react';
+
+export default {
+  title: 'Components/TokenGate',
+  component: TokenGate,
+  parameters: {
+    // TODO: Fix window.ethereum is undefined breaking chromatic
+    chromatic: { disableSnapshot: true },
+  },
+};
+
+const WalletContextTestWrapper = ({ children }): JSX.Element => {
+  const { connected, connectWallet } = useContext(Web3Context);
+
+  return !connected ? <Button onClick={connectWallet}>Connect wallet</Button> : children;
+};
+
+const WithUseWallet = ({ ...props }) => {
+  const { provider } = useWallet();
+
+  return (
+    <>
+      <TokenGate
+        provider={provider!}
+        requiredQuantity={props.requiredQuantity}
+        contractAddress='0x25ed58c027921e14d86380ea2646e3a1b5c55a8b'
+      >
+        <Text>
+          You were able to access this component because you hold at least{' '}
+          {props.requiredQuantity == 0 ? 0 : props.requiredQuantity ? props.requiredQuantity : 1}{' '}
```

Comment:
As far as i understand this statement correct you try to check wether requiredQuantity is undefined? I think you can be simply this statement to something like this:

const text = requiredQuantity === undefined ? 0 : requiredQuantity

Reply:
That's way better. It actually defaults to 1 so i'll do `const text = requiredQuantity === undefined ? 1 : requiredQuantity` but that should work.